Gopkg.toml

```diff
@@ -30,7 +30,7 @@
 
 [[constraint]]
   name = "github.com/eclipse/che-go-jsonrpc"
-  version = "=0.1.0"
+  revision = "931966b891fed8143c920e51e6d4d87b8490b9e8"
```

Comment:
Does it makes sense to release a fresh version of che-go-jsonrpc instead of depending on the revision?

Reply:
It would be nice, if it is not hard...